apps/transaction/templates/transaction/partials/_money_spent_on_room.html

```
@@ -165,34 +165,63 @@ <h5 class="mb-0">{% trans "Who needs to settle up" %}</h5>
+                        {% else %}
+                            <div class="empty-state mt-3">
+                                <p class="fw-semibold mb-1">{% trans "Nothing owed right now" %}</p>
+                                <p class="text-muted small mb-0">{% trans "All participants are even for the tracked expenses." %}</p>
+                            </div>
+                        {% endif %}
+                        {# ── Gross amount covered by others per person ── #}
+                        {% if money_covered_for_person_qs %}
+                            <hr class="my-3">
+                            <p class="text-muted small mb-2">{% trans "Covered by others (gross)" %}</p>
+                            {% for covered in money_covered_for_person_qs %}
+                                {% if covered.total_covered_for_person > 0 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)